.jenkinsci/cancel-builds-same-job.groovy

```diff
@@ -1,3 +1,8 @@
+/*
+ * Copyright D3 Ledger, Inc. All Rights Reserved.
```

Comment:
I think that license information must be added to new files by default. We can do it using `.idea/fileTemplates`. Have you modified this file in the PR? I cannot see, because my browser dies every time I try to open modified files.

Reply:
IDE configurations are not stored in repository.